flax/training/common_utils.py

```diff
@@ -41,6 +41,10 @@ def onehot(labels, num_classes, on_value=1.0, off_value=0.0):
 
 
 def stack_forest(forest):
+  if not forest:
+    return forest
+  if len(forest) == 1:
```

Comment:
Maybe add a comment that this code is needed to work around the limitation of tree_multimap.